pkg/app/piped/cloudprovider/ecs/service.go

```diff
@@ -38,3 +51,12 @@ func parseServiceDefinition(data []byte) (types.Service, error) {
 	}
 	return obj, nil
 }
+
+func parseServiceDefinitionForCluster(data []byte) (string, error) {
+	var obj ServiceDefinition
```

Comment:
nits, looks like this `ServiceDefinition` only be used for parsing 👀  In that case, make this inline type definition would be better
```golang
	var obj struct {
		Cluster string `json:"cluster"`
	}
```